.github/workflows/pluto_build.yml

```diff
@@ -35,44 +35,51 @@ jobs:
       AWS_S3_ENDPOINT: ${{ secrets.AWS_S3_ENDPOINT }}
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-    working-directory: ./db-pluto/pluto_build
     steps:
```

Comment:
I checked in an invalid file to main in my initial commit to get these actions on main - this is just correcting/reverting something that I tried. working-directory can only be specified in steps, not job-wide